doc/components.md

```diff
@@ -6,19 +6,19 @@ The following components groups are included:
 * [MQTT](#mqtt)
 * [Protocol](#protocol)
 * [Services](#services)
+* [Azure Device Registry (ADR) Client*](#azure-device-registry-adr-client)
```

Comment:
nit: I think this was probably a leftover
```suggestion
* [Azure Device Registry (ADR) Client](#azure-device-registry-adr-client)
```

Reply:
wait, also I think this link wouldn't be up here - it's just a sub component of Services and we don't link the other Services

Reply:
Done!